protobuf.bzl

```diff
@@ -193,6 +193,30 @@ def cc_proto_library(
       **kargs)
 
 
+def internal_gen_well_known_protos_java(srcs):
+  """Bazel rule to generate the gen_well_known_protos_java genrule
+
+  Args:
+    srcs: the well known protos
+  """
+  root = Label("%s//protobuf_java" % (REPOSITORY_NAME)).workspace_root
```

Comment:
Looks good. I can `bazel build //:protobuf_java` in the protobuf repo and I can depend on the repo, e.g., `git_repository(name = 'protobuf' ...)` and then either use `@protobuf//:protobuf_java` or bind them and use `//external:protobuf_java` and everything works. I've tested against `protobuf_java`, `protoc`, and `protoc_lib` targets.

Anything else you want me to test?